R/ws.R

```diff
@@ -155,6 +155,7 @@ style_file <- function(path,
 #' @inheritParams style_dir
 #' @param path The path to a file that should be styled.
 prettify_one <- function(transformers, path) {
-  if (!grepl("\\.[Rr]$", path)) stop(path, " is not a .R file")
+  if (!grepl("\\.[r](md)?$", path, ignore.case = TRUE))
+    stop(path, " is not an R or Rmd file")
```

Comment:
Do we still need this check if we check further downstream?

Reply:
You're right the downstream check does make this redundant. The downstream check has to be there since it provides the logic for R/Rmd handling. I suppose the only advantage to having this earlier check is that the process is stopped before the call to `transform_files()` which is before any `messages` are output to the console.

I'll check if it can be removed.

Reply:
Thanks. In that case, you can remove `prettify_one()`.